src/docs/asciidoc/reference_doc/supervisor.adoc

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2023 RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// This document is subject to the terms of the Creative Commons Attribution 4.0 International license.
+// If a copy of the license was not distributed with this
+// file, You can obtain one at https://creativecommons.org/licenses/by/4.0/.
+// SPDX-License-Identifier: CC-BY-4.0
+
+
+= Supervisor
+Supervisor service can monitor users connections and card acknowledgements to send alert cards when specified conditions are met.
+
+== Entities connection
+It is possible to configure supervisor module for monitoring users connections to Opfab to ensure that there is at least one user connected to Opfab for each configured entity. When no user of an entity connects to Opfab for a specified period, supervisor module will send an alert card to the configured recipient entity.
+
+== Card acknowledgement
+It is possible to configure supervisor module to monitor card acknowledgements for specific processes/states. When a card of a configured process and state remains not acknowledged for a configured period, supervisor module will send an alert to the card recipients that didn’t acknowledge the card.
```

Comment:
will send an alert to the card **publisher with the list of recipients that did not acknowledge the card**